client/components/foldable-card/index.jsx

```diff
@@ -41,7 +41,7 @@ var FoldableCard = React.createClass( {
 			onClose: noop,
 			cardKey: '',
 			icon: 'chevron-down',
-			isExpanded: false
+			isExpanded: false,
```

Comment:
Typo, since I had an extra prop defined at first, which I removed. I'll remove the comma here.